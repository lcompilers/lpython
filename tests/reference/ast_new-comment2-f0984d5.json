{
    "basename": "ast_new-comment2-f0984d5",
    "cmd": "lpython --show-ast --new-parser --no-color {infile} -o {outfile}",
    "infile": "tests/tokens/comment2.py",
    "infile_hash": "b62404a5d774f0b18fcad6fb1dac8733334ce3593a77d34bcbb05019",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_new-comment2-f0984d5.stdout",
    "stdout_hash": "2a109027d057438f0517cc77b3ce793a3f5f35574c09aec9862cd247",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}