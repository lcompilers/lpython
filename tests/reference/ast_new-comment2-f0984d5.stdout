(Module
    [(FunctionDef
        main
        ([]
        []
        []
        []
        []
        []
        [])
        [(If
            (Name
                foo
                Load
            )
            [(If
                (Name
                    bar
                    Load
                )
                [(Pass)]
                []
            )]
            [(Pass)]
        )
        (Return
            ()
        )]
        []
        ()
        ()
    )
    (FunctionDef
        test
        ([]
        []
        []
        []
        []
        []
        [])
        [(Expr
            (Call
                (Name
                    print
                    Load
                )
                []
                []
            )
        )]
        []
        ()
        ()
    )
    (FunctionDef
        main
        ([]
        []
        []
        []
        []
        []
        [])
        [(Expr
            (Call
                (Name
                    print
                    Load
                )
                []
                []
            )
        )]
        []
        ()
        ()
    )
    (FunctionDef
        main
        ([]
        []
        []
        []
        []
        []
        [])
        [(Expr
            (Call
                (Name
                    print
                    Load
                )
                []
                []
            )
        )
        (Pass)]
        []
        ()
        ()
    )
    (FunctionDef
        main
        ([]
        []
        []
        []
        []
        []
        [])
        [(If
            (Name
                foo
                Load
            )
            [(Pass)
            (Pass)
            (If
                (Name
                    bar
                    Load
                )
                [(Pass)]
                []
            )
            (Pass)]
            []
        )
        (Pass)]
        []
        ()
        ()
    )
    (Assign
        [(Name
            x
            Store
        )]
        (List
            [(Name
                a
                Load
            )]
            Load
        )
        ()
    )]
    []
)
